### ISSUE TITLE
PanelBar: Elements with role "menuitem" are not expected to have "aria-selected" attribute

### ISSUE THREAD

**Issue by ylux587** [telerik/kendo-ui-core#5900]
### Reproduction of the problem
Not allowed ARIA attribute found in panel bar control's html

### Current behavior
Rendering panel bar generated not allowed aria attribute:  aria-selected="true"

### Expected/desired behavior
ARIA attribute is not allowed: aria-selected="true"

### Environment

* **Kendo UI version:** Kendo UI version: 2019.3.1114
* **jQuery version:** 3.3.28
* **Browser:**Chromium Edge Version: 84.0.488.1 (Official build) dev (64-bit)

**Comment by dmanova** [comment]
@ylux587 could you specify the exact tags where aria-selected is used but is not allowed? And the rule/ section that describes its usage is not allowed.?

I see that aria-selected="true" property is used in < li > tags e.g.:
```
<li class="k-item k-state-default k-first" role="menuitem" aria-selected="true">
    <span class="k-link k-state-selected">Q1 Forecast</span>
</li>
```
And this property is read by some of the readers like ChromeVox, although not considered by others like Narrator.

**Comment by dmanova** [comment]
I managed to follow the mentioned issue to WCAG 2.1 related technical specification:
https://www.w3.org/TR/wai-aria-1.1/#menuitem
According to it, the more appropriate property seems to be _aria-current_

**Comment by ylux587** [comment]
Hi, @dmanova ,

Thanks for your previous reply. After double check I think the problem should be described as 

ARIA attribute is not allowed: aria-selected="true"

I think your suggestion to use aria-current is right fix for it. Could you please suggest how to do it or when you will provide a fix on it?

Thanks!

**Comment by dmanova** [comment]
Hi, @ylux587 . Until a fix is provied you could add the aria-current attribute with a code similar to this one: 

`      var onSelect = function(e) {`
`        console.log("selected");`
`        $("li[aria-current='true']" ).removeAttr("aria-current");`
`        $("li[aria-selected='true']" ).attr("aria-current","true");`
`        //$("li[aria-current='true']" ).removeAttr("aria-selected"); //uncomment this row if you want to remove the aria-selected attribute`
    };      
`     $("#panelbar").data("kendoPanelBar").bind("select", onSelect);`

**Comment by Dimitar-Goshev** [comment]
@ylux587 The correct role for Kendo UI PanelBar is tree and we are applying it for R3 2020 SP1.
